tests/Makefile

```diff
@@ -1,24 +1,40 @@
 
 all: setup test teardown
-	
+
 
 test:
-	docker exec -t tests_cstar_1 cstar run --command='nodetool status' --seed-host cassandra-0 --ssh-password=root
+	docker exec -t tests_cstar_1 bash -c "cd /usr/src/app/cstar/ && behave tests/features -D SEED_HOST=cassandra-10 --no-capture --no-capture-stderr --format=plain --junit --junit-directory=/usr/src/app/cstar/tests/results"
```

Comment:
@adejanovski that's the part I was missing/not knowing how to plug! 

Thanks for taking over on this @adejanovski (LGTM)